src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -1053,6 +1125,7 @@ static Vector64<uint> SoftwareFallback(uint e0, uint e1)
         /// <summary>Creates a new <see cref="Vector64{Byte}" /> instance with the first element initialized to the specified value and the remaining elements initialized to zero.</summary>
         /// <param name="value">The value that element 0 will be initialized to.</param>
         /// <returns>A new <see cref="Vector64{Byte}" /> instance with the first element initialized to <paramref name="value"/> and the remaining elements initialized to zero.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I assume these additions were intentional?

Reply:
Yes, they already existed for Vector128/256 but were missed for Vector64 in a previous PR.